packages/vm/tests/api/runTx.spec.ts

```diff
@@ -559,3 +560,29 @@ tape('runTx() -> consensus bugs', async (t) => {
     t.end()
   })
 })
+
+tape('runTx: RunTxOptions', (t) => {
```

Comment:
nit to match above naming

```suggestion
tape('runTx() -> RunTxOptions', (t) => {
```